docs/en/operations/settings/permissions-for-queries.md

```diff
@@ -16,28 +16,28 @@ Queries in ClickHouse can be divided into several types:
 
 The following settings regulate user permissions by the type of query:
 
--   [readonly](#settings_readonly) — Restricts permissions for all types of queries except DDL queries.
+-   [readonly](#settings_readonly) — Restricts permissions for Write and DDL queries, and selectively on Change settings queries.
```

Comment:
> should I also say something about DELETE and UPDATE in here?

It's much easier to enumerate queries that are allowed. I would say that it restricts permissions to all types of queries except readonly queries (like SELECT and equivalent queries), and queries that modify only session context (like USE  and (for readonly=2) SET and `CREATE TEMPORARY TABLE`). Queries like EXISTS, DESCRIBE, `SHOW PROCESSLIST`, etc are equivalent to SELECT, because they just do select from system tables. EXPLAIN is also equivalent to SELECT.